app/services/witter.js

```diff
@@ -0,0 +1,14 @@
+const axios = require('axios').default;
+const logger = require('../logger');
+
+const WitterInstance = axios.create({
+  baseURL: 'http://numbersapi.com'
+});
+
+exports.getWitter = (num = Math.floor(Math.random() * 100)) =>
```

Comment:
This is fine, but try to avoid putting complex stuff on the args literals, create a constant for this random number and assign it as default, so it's easier to read